.github/workflows/build-and-test-arm.yaml

```
@@ -6,7 +6,15 @@ on:
   workflow_dispatch:
 
 jobs:
+  pre_condition:
+    name: Don't run scheduled jobs on forks
```

We don't block all scheduled jobs.

```suggestion
    name: Prevent running on forks
```